<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsDetailed</class>
 <widget class="QWidget" name="OptionsDetailed">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1,0,0,0,0,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Format:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="formatComboBox"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="pluginLabel">
       <property name="text">
        <string>Use Plugin:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="pluginComboBox"/>
     </item>
     <item>
      <widget class="QToolButton" name="configurePluginButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="configure">
         <normaloff/>
        </iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="pluginLayout"/>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="filterLayout"/>
   </item>
   <item>
    <widget class="Line" name="filterLine">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="OutputDirectory" name="outputDirectory" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="replayGainCheckBox">
       <property name="text">
        <string>Calculate Replay Gain tags</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="replayGainSpinBox">
       <property name="suffix">
        <string> dB</string>
       </property>
       <property name="minimum">
        <double>-99.000000000000000</double>
       </property>
       <property name="maximum">
        <double>99.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="configureReplayGainButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="configure"/>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="saveProfileButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="document-save">
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="loadProfileButton">
       <property name="toolTip">
        <string>Load saved profiles</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="document-open">
         <normaloff/>
        </iconset>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OutputDirectory</class>
   <extends>QWidget</extends>
   <header>outputdirectory.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
